collectors/property.yaml

```diff
@@ -22,8 +22,22 @@ HostSystemPropertiesCollector:
       - label: "runtime_memory_capacity_kilobytes"
         property: "runtime|memoryCap"
 
+VMPropertiesCollector:
+    enum_metrics:
+      - label: "runtime_powerState"
+        expected: "Powered On"
+        property: "summary|runtime|powerState"
+      - label: "runtime_connectionState"
+        expected: "connected"
+        property: "summary|runtime|connectionState"
 
 
+    info_metrics:
+#      - label: "virtualdisk_datastore"
+#        property: "virtualDisk|Datastore"
```

Comment:
please add a comment, that they are not yet enabled on vrops side. Same for the other, disabled metrics.

Reply:
This has been taken care that mertics was not returning any value, so i have removed it for now. -Thanks